classes/Policies.php

```diff
@@ -234,12 +250,19 @@ class Policies {
      */
     public function getArray() {
         $out = array();
-        foreach ($this->policies as $policy_id => $policy_text)
-        {
+        if ( $this->policy_id ) {
             $out[] = array(
-                $policy_id,
-                $policy_text
+                $this->policy_id,
+                $this->policies[$this->policy_id]
```

Comment:
Should this throw an exception if policy_id is invalid? (Or should it do that in the constructor?)